src/components/app/App.js

```diff
@@ -9,14 +16,19 @@ import { Header } from '../header/HeaderFromLecture';
 import './App.css';
 import 'bootstrap/dist/css/bootstrap.min.css';
 
+
 function App() {
-  return (
+    return (
     <div className="App">
       <Header />
       <div>content will appear here soon</div>
-      {/*  todo: здесь нужно использовать Footer по аналогии с Header */}
 
-      {/*  todo: здесь нужно использовать UserCard по аналогии с Header, как пропсу user передать usersList[0]*/}
+        {<Footer/>
```

Comment:
you don't need to put JSX components inside curly brackets {}
So you can just leave 
```
<UserCard user={usersList}/>
<Footer/>
```